docs/modules/deployment/pages/time-series-storage/timeseries/configuration.adoc

```diff
@@ -6,27 +6,27 @@ Set the following property in `$\{OPENNMS_HOME}/etc/opennms.properties` to confi
 [source, properties]
 org.opennms.timeseries.strategy=integration
 
-After activating the Time Series Integration Layer, you must implement it; this is done via the Karaf shell:
+[NOTE]
+====
+If you are using a container, you can set this as an environment variable in your container's configuration settings instead of creating or modifying a property file:
 
-[source, console]
-----
-clone & build:
-> git clone git@github.com:opennms-forge/timeseries-integration-inmemory.git
-> mvn install
-
-in Karaf shell:
-> bundle:install -s mvn:org.opennms.plugins.timeseries.inmemory/timeseries-inmemory-plugin/1.0.0-SNAPSHOT
-----
-
-For specific instructions, check your plugin description.
+[source, properties]
+OPENNMS_TIMESERIES_STRATEGY=integration
+====
 
+[TIP]
+====
 We also highly recommend that you reference resources stored in the Time Series Integration Layer by their foreign source and foreign ID, as opposed to their database ID.
 To this end, set the following property in the same file:
```

Comment:
Which file does this refer to? In the note above, we're referring to the container's configuration settings.

Reply:
any `opennms.properties.d` file. potentially the same file referenced in line 6, which makes me realize that line 4 references editing the main properties files and not creating a new one.